src/com_tjvendors/site/includes/vendor.php

```diff
@@ -430,4 +511,115 @@ public function getPaymentConfig()
 	{
 		return !empty($this->payment_gateway) ? new Registry($this->payment_gateway) : false;
 	}
+
+	/**
+	 * Get the vendor address
+	 *
+	 * @return  String
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function getAddress()
+	{
+		return $this->address;
+	}
+
+	/**
+	 * Get the vendor other city
+	 *
+	 * @return  String
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function getOtherCity()
+	{
+		return $this->other_city;
+	}
+
+	/**
+	 * Get the vendor zip
+	 *
+	 * @return  String
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function getZip()
+	{
+		return $this->zip;
+	}
+
+	/**
+	 * Get the vendor phone number
+	 *
+	 * @return  String
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function getPhoneNumber()
+	{
+		return $this->phone_number;
+	}
+
+	/**
+	 * Get the vendor website address
+	 *
+	 * @return  String
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function getWebsiteAddress()
+	{
+		return $this->website_address;
+	}
+
+	/**
+	 * Get the vendor vat number
+	 *
+	 * @return  String
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function getVatNumber()
+	{
+		return $this->vat_number;
+	}
+
+	/**
+	 * This function return vendor profile complete/incomplete status
+	 *
+	 * @param   Integer  $userId  userId
+	 * @param   Integer  $client  client
+	 * 
+	 * @return  boolean  true or false
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function getVendorProfileStatus($userId, $client)
```

Comment:
this should return number not true / false

Reply:
Resolved